app/controllers/ApplicationController.scala

```diff
@@ -123,10 +123,11 @@ class ApplicationController @Inject()(val applicationService: ApplicationService
           }
 
           def asSeqOfSeq(data: ApiData): Seq[(String, Seq[(ApiVersion, ApiStatus)])] = {
+
             if(data.versions.isEmpty) {
               Seq.empty
             } else {
-              Seq( (data.name, data.versions.toSeq.map(v => (v._1, v._2.status))) ) 
+              Seq( (data.name, data.versions.toSeq.sortBy(v => v._1).map(v => (v._1, v._2.status))) )
```

Comment:
Are you sure you don't want to create an implicit Ordering for this and just use the .sorted method.

Reply:
We didn't think it would be reused.